src/Filter/StringFilter.php

```diff
@@ -52,14 +56,20 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
         if (ChoiceType::TYPE_EQUAL == $data['type']) {
             $queryBuilder->setParameter($parameterName, $data['value']);
```

Comment:
Do you have an alternative?
We could mention it at least in the docs and the info()

Reply:
doing it in php, with `mb_strtolower`? I don't think I fully understood the issue yet, sorry. For instance, what are you referring to with `EQUAL`?

Reply:
I think there is another issue on this repo where I refuse to have this be the default because of performance implications… but I guess it is fine if people add a function based index. Maybe this is what you should add in the `info()` call: https://use-the-index-luke.com/sql/where-clause/functions/case-insensitive-search

Reply:
Done [here](https://github.com/sonata-project/SonataAdminBundle/pull/5353/files#diff-978c02ccc8ccdf14cb8a58dc91ab346c), is it what you expect @greg0ire ?